great_expectations/expectations/metrics/query_metrics/query_column.py

```diff
@@ -0,0 +1,57 @@
+from typing import Any, Dict, Optional, Union
+
+from great_expectations.execution_engine import (
+    ExecutionEngine,
+    PandasExecutionEngine,
+    SparkDFExecutionEngine,
+    SqlAlchemyExecutionEngine,
+)
+from great_expectations.execution_engine.execution_engine import MetricDomainTypes
+from great_expectations.expectations.metrics.import_manager import sa
+from great_expectations.expectations.metrics.metric_provider import metric_value
+from great_expectations.expectations.metrics.query_metric_provider import (
+    QueryMetricProvider,
+)
+
+
+class QueryColumn(QueryMetricProvider):
+    metric_name = "query.column"
+    value_keys = (
+        "column",
+        "query",
+    )
+
+    @metric_value(engine=SqlAlchemyExecutionEngine)
+    def _sqlalchemy(
+        cls,
+        execution_engine: SqlAlchemyExecutionEngine,
+        metric_domain_kwargs: Dict,
+        metric_value_kwargs: Dict,
```

Comment:
The type hints here should either specify the contents of `Dict` or simply use `dict` (please check how it is done in other metric implementation functions).

Reply:
Do we want to fix this in extant metrics as well? At a glance it looks like at least all Table metrics use the above `Dict` pattern w/ no type hints.

Reply:
@austiezr just worry about this metric and we can cleanup the others at a later date.